### ISSUE TITLE
Bug: hardcoded nordvpn server list url invalid

### ISSUE THREAD

**Issue by ralandnet** [qdm12/gluetun#1698]
### Is this urgent?

Yes

### Host OS

Synology

### CPU arch

x86_64

### VPN service provider

NordVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

v3

### What's the problem 🤔

\internal\provider\nordvpn\updater\api.go

invalid url:
- https://api.nordvpn.com/v1/servers/
- {"message":"Not Found"}

correct url:
- https://api.nordvpn.com/server

### Share your logs

```plain text
gluetun    | 2023-06-17T23:45:36+02:00 INFO [storage] merging by most recent 13056 hardcoded servers and 13056 servers read from /gluetun/servers.json
gluetun    | 2023-06-17T23:45:36+02:00 ERROR VPN settings: provider settings: server selection: the hostname specified is not valid: value is not one of the possible choices: value "it267.nordvpn.com", choices available are al18.nordvpn.com, al20.nordvpn.com, ....
```

**Comment by qdm12** [comment]
Indeed, fixed in 515ae8efb3f9af8427cef933bb29154f8a80ced9 👍

**Comment by matlinski** [comment]
Hello I have a similar issue but with countries and cities. 

environment variables:
```
VPN_SERVICE_PROVIDER=nordvpn
OPENVPN_USER=secret
OPENVPN_PASSWORD=secret
SERVER_COUNTRIES=Germany
```

logs:
```
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version latest built on 2023-03-01T14:10:28.105Z (commit ab5dbdc)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-06-20T10:55:22Z INFO [routing] default route found: interface eth0, gateway 192.168.0.1 and assigned IP 192.168.128.3
2023-06-20T10:55:22Z INFO [routing] local ethernet link found: eth0
2023-06-20T10:55:22Z INFO [routing] local ipnet found: 192.168.0.0/16
2023-06-20T10:55:22Z INFO [firewall] enabling...
2023-06-20T10:55:22Z INFO [firewall] enabled successfully
2023-06-20T10:55:23Z INFO [storage] creating /gluetun/servers.json with 13184 hardcoded servers
2023-06-20T10:55:24Z ERROR VPN settings: provider settings: server selection: the country specified is not valid: one or more values is set but there is no possible value available
2023-06-20T10:55:24Z INFO Shutdown successful
```

It works if I pass domain or a domain list, but countries or cities always fail, tried everything.
